### PR TITLE
Conda pkg: reformat README; add a bit on how tag is constructed

### PR DESCRIPTION
**Pull request by vladsaveliev**
This README will be referenced in the team-docs here: https://github.com/populationgenomics/team-docs/pull/48 and here: https://github.com/populationgenomics/team-docs/blob/main/getting_started.md#hail

### CODE REVIEW

File: conda/README.md [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
-that have not been released yet, or even use features that will not be
-propagated to the upstream repository at all.
+synced with the [official PyPI release](https://pypi.org/project/hail). However, having
+a separate conda package in the `cpg` channel allows us to build it against the codebase
+in our fork.
 
-[GitHub Actions CI](../.github/workflows/main.yaml) is set up to build the package using this recipe and push it to Anaconda on every push to the `main` branch in the [CPG hail fork](https://github.com/populationgenomics/hail).
+Because we don't control versioning of Hail project, the `cpg` conda package is
+versioned specifically: we append the git commit hash to the official version tag,
+e.g. `0.2.62.dev289c163`.
```
Comment (illusional) [inline conda/README.md:14]:
```suggestion
We don't control versioning of original Hail project, so our `cpg` conda release name 
is the official version tag appended with the git commit has, e.g. `0.2.62.dev289c163`.
```

File: conda/README.md [hunk 2]
```diff
@@ -1,19 +1,36 @@
...
 
-To install the package, set up miniconda first:
+[GitHub Actions CI](../.github/workflows/condarise.yaml) is set up to build the package
+using this recipe and push it to Anaconda on every push event to the `main` branch in
+the
+[CPG hail fork](https://github.com/populationgenomics/hail).
+
+When installing the package, list the `cpg` channel before `bioconda` to prioritize it
+in the channel order:
```
Comment (illusional) [inline conda/README.md:22]:
```suggestion
When installing the package, list the `cpg` channel before `bioconda` to prioritize
the channel order:
```

**Review by illusional — APPROVED** [review]
Nice! I've added some small suggestions but approved :)

**Comment by vladsaveliev** [comment]
Ah, forgot to merge. Thank you for the edits! :)

**Pull request merged**